### PR TITLE
Add postgres

### PR DESCRIPTION
**Pull request by jonmattgray**
Add postgres boilerplate and first migration (attachments table). Includes helm config for db

### CODE REVIEW

File: docs/db.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+`attachments` is a staging ground for files before they're uploaded to IPFS as part of on-chain token creation.
+
+#### Columns
+
+| column        | PostgreSQL type           | nullable |       default        | description                            |
+| :------------ | :------------------------ | :------- | :------------------: | :------------------------------------- |
+| `id`          | `UUID`                    | FALSE    | `uuid_generate_v4()` | Unique identifier for the `attachment` |
+| `binary_blob` | `bytea`                   | FALSE    |          -           | Attachment file data as binary         |
+| `created_at`  | `Timestamp with timezone` | FALSE    |       `now()`        | When the row was first created         |
```
Comment (mattdean-digicatapult) [inline docs/db.md:27]: probably want filename in here as well

**Review by mattdean-digicatapult — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@digicatapult/inteli-api",
-  "version": "1.1.0",
+  "version": "1.2.0",
   "description": "Insert repo description",
   "main": "app/index.js",
   "scripts": {
```
Comment (danielcooperxyz) [inline package.json:6]: do we need to define a migration script here?

**Review by danielcooperxyz — APPROVED** [review]
Good PR.

I have one question/comment about whether we need a migration script in package.json, but I'd be happy to merge without it! I see there is a readme addition so possibly not important to add the script in package.json

File: package.json (hunk 2 above)
Comment (jonmattgray) [reply package.json:6]: something like `"migrate": "knex migrate:latest"`? That would be nice, could add in next PR to save re-review here

**Pull request merged**